test-suite/src/function/current_date.rs

```diff
@@ -0,0 +1,35 @@
+use {
+    crate::*,
+    gluesql_core::prelude::{Payload, Value::*},
+};
+
+test_case!(current_date, {
+    let g = get_tester!();
+
+    macro_rules! date {
+        ($date: expr) => {
+            $date.parse().unwrap()
+        };
+    }
+
+    let test_cases = [
```

Comment:
could we try
ref. https://github.com/gluesql/gluesql/blob/main/test-suite/src/function/slice.rs

```rust
g.named_test
g.test
```
patterns?

we are trying to replace legacy test templates to use `g.test` or `g.named_test`
(`g.test` for simple unnamed, and we can specify test case name using `g.named_test`)